jupyter-js-widgets/css/widgets-base.css

```diff
@@ -717,19 +717,33 @@ ul.widget-dropdown-droplist.mod-active {
     border: var(--jp-widgets-input-border-width) solid var(--jp-widgets-input-border-color);
     border-radius: 0;
     height: var(--jp-widgets-inline-height);
-    line-height: var(--jp-widgets-inline-height);
     flex: 1 1 var(--jp-widgets-inline-width-short);
-    background-color: var(--jp-widgets-input-backgroundColor);
+    background-color: var(--jp-widgets-input-background-color);
 }
 
 .widget-select select {
+    outline: none !important;
```

Comment:
Again, this is a huge accessibility problem. We need to have some way to signify the slider handle has focus for keyboard input.